src/util/nettest.js

```diff
@@ -50,103 +50,6642 @@ export const getDeckIcon = (iconName, other) => {
+  "mou": "US", 
+  "mov": "AU", 
+  "moz": "PF", 
+  "mpa": "NA", 
+  "mpc": "ID", 
+  "mpd": "PK", 
+  "mph": "PH", 
+  "mpk": "KR", 
+  "mpl": "FR", 
+  "mpm": "MZ", 
+  "mpn": "FK", 
+  "mpt": "TL", 
+  "mpw": "UA", 
+  "mpy": "GF", 
+  "mqa": "AU", 
+  "mqc": "PM", 
+  "mqd": "AR", 
+  "mqe": "AU", 
+  "mqf": "RU", 
+  "mqh": "BR", 
+  "mqk": "BO", 
+  "mql": "AU", 
+  "mqm": "TR", 
+  "mqn": "NO", 
+  "mqp": "ZA", 
+  "mqq": "TD", 
+  "mqs": "VC", 
+  "mqu": "CO", 
+  "mqx": "ET", 
+  "mqz": "AU", 
+  "mra": "LY", 
+  "mrd": "VE", 
+  "mre": "KE", 
+  "mrg": "AU", 
+  "mrh": "GB", 
+  "mri": "US", 
+  "mro": "NZ", 
+  "mrp": "AU", 
+  "mrq": "PH", 
+  "mrr": "EC", 
+  "mrs": "FR", 
+  "mru": "MU", 
+  "mrv": "RU", 
+  "mrw": "DK", 
+  "mrx": "IR", 
+  "mry": "US", 
+  "mrz": "AU", 
+  "msa": "CA", 
+  "mse": "GB", 
+  "msh": "OM", 
+  "msi": "ID", 
+  "msj": "JP", 
+  "msl": "US", 
+  "msm": "CD", 
+  "msn": "US", 
+  "mso": "US", 
+  "msp": "US", 
+  "msq": "BY", 
+  "msr": "TR", 
+  "mss": "US", 
+  "mst": "NL", 
+  "msu": "LS", 
+  "msw": "ER", 
+  "msx": "CG", 
+  "msy": "US", 
+  "msz": "AO", 
+  "mta": "NZ", 
+  "mtb": "CO", 
+  "mtd": "AU", 
+  "mte": "BR", 
+  "mtf": "ET", 
+  "mti": "CV", 
+  "mtk": "KI", 
+  "mtl": "AU", 
+  "mtq": "AU", 
+  "mtr": "CO", 
+  "mts": "SZ", 
+  "mtt": "MX", 
+  "mtu": "MZ", 
+  "mtv": "VU", 
+  "mtw": "ID", 
+  "mty": "MX", 
+  "mtz": "IL", 
+  "mua": "SB", 
+  "mub": "BW", 
+  "muc": "DE", 
+  "mud": "MZ", 
+  "mue": "US", 
+  "muf": "ID", 
+  "mug": "MX", 
+  "muh": "EG", 
+  "muk": "CK", 
+  "mum": "KE", 
+  "mun": "VE", 
+  "mup": "AU", 
+  "muq": "AU", 
+  "mur": "MY", 
+  "mus": "JP", 
+  "muw": "DZ", 
+  "mux": "PK", 
+  "muy": "CG", 
+  "muz": "TZ", 
+  "mva": "IS", 
+  "mvb": "GA", 
+  "mvd": "UY", 
+  "mvf": "BR", 
+  "mvk": "AU", 
+  "mvo": "TD", 
+  "mvp": "CO", 
+  "mvq": "BY", 
+  "mvr": "CM", 
+  "mvs": "BR", 
+  "mvt": "PF", 
+  "mvv": "FR", 
+  "mvx": "GA", 
+  "mvy": "US", 
+  "mvz": "ZW", 
+  "mwb": "AU", 
+  "mwd": "PK", 
+  "mwe": "SD", 
+  "mwf": "VU", 
+  "mwk": "ID", 
+  "mwn": "TZ", 
+  "mwp": "NP", 
+  "mwq": "MM", 
+  "mwy": "AU", 
+  "mwz": "TZ", 
+  "mxb": "ID", 
+  "mxd": "AU", 
+  "mxh": "PG", 
+  "mxi": "PH", 
+  "mxj": "NG", 
+  "mxl": "MX", 
+  "mxm": "MG", 
+  "mxn": "FR", 
+  "mxp": "IT", 
+  "mxs": "WS", 
+  "mxt": "MG", 
+  "mxu": "AU", 
+  "mxv": "MN", 
+  "mxw": "MN", 
+  "mxx": "SE", 
+  "mxz": "CN", 
+  "mya": "AU", 
+  "myb": "GA", 
+  "myc": "VE", 
+  "myd": "KE", 
+  "mye": "JP", 
+  "myg": "BS", 
+  "myi": "AU", 
+  "myj": "JP", 
+  "mym": "GY", 
+  "myn": "YE", 
+  "myo": "AU", 
+  "myp": "TM", 
+  "myq": "IN", 
+  "myr": "US", 
+  "myt": "MM", 
+  "myu": "US", 
+  "myw": "TZ", 
+  "myy": "MY", 
+  "myz": "MW", 
+  "mzb": "MZ", 
+  "mzc": "GA", 
+  "mzd": "EC", 
+  "mzg": "TW", 
+  "mzh": "TR", 
+  "mzi": "ML", 
+  "mzk": "KI", 
+  "mzl": "CO", 
+  "mzm": "FR", 
+  "mzo": "CU", 
+  "mzp": "NZ", 
+  "mzq": "ZA", 
+  "mzr": "AF", 
+  "mzs": "MY
```

Comment:
If you are to add such a big dump of the airport codes maybe we can embed it in a smarter way, by building the map from the countrycode->airport map?